x-pack/plugins/security_solution/public/management/pages/policy/store/policy_details/reducer/trusted_apps_reducer.ts

```diff
@@ -14,6 +14,35 @@ export const policyTrustedAppsReducer: ImmutableReducer<PolicyDetailsState, AppA
   state = initialPolicyDetailsState(),
   action
 ) => {
-  // FIXME: implement trusted apps reducer
+  if (action.type === 'policyArtifactsAvailableListPageDataChanged') {
```

Comment:
This is the part I mean there's no business logic in these reducers

Reply:
Are you talking about the test cases?

Reply:
yes inded.